packages/transactions/src/builders.ts

```diff
@@ -630,16 +643,9 @@ export interface SignedTokenTransferOptions extends TokenTransferOptions {
   senderKey: string;
 }
 
-export interface UnsignedMultiSigTokenTransferOptions extends TokenTransferOptions {
-  numSignatures: number;
-  publicKeys: string[];
-}
+export type UnsignedMultiSigTokenTransferOptions = TokenTransferOptions & UnsignedMultiSigOptions;
```

Comment:
Refactored these types to be more reusable. Do you think the change from `interface` to `type` is a problem?